### ISSUE TITLE
Possible integer overflow when calculating feed expiration

### ISSUE THREAD

**Issue by abitmore** [bitshares/bitshares-core#888]
[Code](https://github.com/bitshares/bitshares-core/blob/2.0.180425/libraries/chain/include/graphene/chain/asset_object.hpp#L211-L212):
```
         time_point_sec feed_expiration_time()const
         { return current_feed_publication_time + options.feed_lifetime_sec; }
```

Update:
Actually we don't need a hard fork to fix this issue: if overflow, `feed_expiration_time()` will return a smaller result, so in worst cases we will update median and check call orders when don't need to do so, but actually nothing will change in database.

**Comment by abitmore** [comment]
Not a hard fork bug. OP updated.

**Comment by abitmore** [comment]
Fixed with #954.
